GettingStarted.md

```diff
@@ -0,0 +1,126 @@
+# Getting Started
+
+This SDK has several entry points:
+
+- You can run the application using the [CLI](./src/dotnet//CarbonAware.CLI).
+
+- You can build a container containing the [WebAPI](./src/dotnet/CarbonAware.WebApi) and connect via REST requests.
+
+- (Future) You can install the Nuget package and make requests directly.
+
+Each of these has configuration requirements which are detailed below.
+
+## Data Sources
+
+We intend to support multiple data sources for carbon data.  At this time, only a JSON file and [WattTime](https://www.watttime.org/) are supported.  To use WattTime data, you'll need to acquire a license from them and set the appropriate configuration information.
+
+## Configuration
+
+This project uses standard [Microsoft.Extensions.Configuration](https://docs.microsoft.com/en-us/dotnet/core/extensions/configuration) mechanisms.
+
+The WebAPI project uses standard configuration sources provided by [ASPNetCore](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/).  Please review this link to understand how configuration is loaded and the priority of that configuration.
+
+Please note that configuration is heirarchical.  The last configuration source loaded that contains a configuration value will be the value that's used.  This means that if the same configuration value is found in both appsettings.json and as an environment variable, the value from the environment variable will be the value that's applied.
+
+When adding values via environment variables, we recommend that you use the double underscore form, rather than the colon form.  Colons won't work in non-windows environment.  For example:
+
+```bash
+  CarbonAwareVars__CarbonIntensityDataSource="WattTime"
+```
+
+Note that double underscores are used to represent dotted notation or child elements that you see in the JSON below.  For example, to set proxy information using environment variables, you'd do this:
+
+```bash
+  CarbonAwareVars__Proxy__UseProxy
+```
+
```

Comment:
Suggestion: Add a complete set of env vars (aiming for a AKS env or run it on vscode) needed in order to go through the proxy and hit watttime data src.

Reply:
Good idea.  I'll add it.